### DISCUSSION TITLE
Why is it that "invoke-cpu" profile is commented out?

### DISCUSSION THREAD

**Discussion by devingfx** [AbdBarho/stable-diffusion-webui-docker#607] [General]
I'm a bit sad as a "CPU only" user wanting to use invokeAI :crying_cat_face:

**Comment by AbdBarho** [comment]
Apologies, it is not ready yet, I am now working on an update to invoke 3.3.0
I will see if I can make the CPU version running.

**Comment by devingfx** [comment]
Thx a lot for your work !
Don't hesitate to post here any update for me to test it on my potato PC ;)
